CHANGELOG.md

```diff
@@ -8,8 +8,76 @@
 #### 🐛 Type: Bug
 
 - `permissions`
+
   - [#1259](https://github.com/commercetools/merchant-center-application-kit/pull/1259) fix(permissions): data fences to be overwritten by manage resource permission ([@tdeekens](https://github.com/tdeekens))
 
+Note that in this release we have added log statements whenever we encounter an what we consider an not recommended use of our permission components. The permission components include `RestrictedByPermissions`, `injectAuthorized` and `useIsAuthorized`.
```

Comment:
@emmenko this is what you had in mind, right?

Reply:
Yes, that's the idea. Thanks!